zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -864,6 +860,17 @@ public static int getSnapCount() {
         }
     }
 
+    public int getGlobalOutstandingLimit() {
+        String sc = System.getProperty(GLOBAL_OUTSTANDING_LIMIT);
```

Comment:
are we reading the system property and parsing the value at every call of shouldThrottle ?
can't we cache the value ?